src/project.rs

```diff
@@ -73,6 +75,15 @@ pub struct Project {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub serve_address: Option<IpAddr>,
 
+    /// The mode to use when mapping scripts into Roblox.
+    /// Can be either `Class` or `RunContext` and determines whether script
```

Comment:
Missed this in my earlier review but this comment needs to be updated to reflect that it's a bool and not an enum now.

Reply:
*Just* committed before I saw that lol - let me fix